.travis.yml

```diff
@@ -28,8 +28,8 @@ addons:
 bundler_args: --without test --jobs 3 --retry 3
 before_install:
   - "rm ${BUNDLE_GEMFILE}.lock"
-  - "travis_retry gem update --system"
-  - "travis_retry gem install bundler -v 1.15.4"
+  - "travis_retry gem update --system 2.6.14"
+  - "travis_retry gem install bundler"
```

Comment:
Is it intended to remove the bundler version?

Reply:
At least doesn't pass testing https://github.com/rails/rails/tree/master/guides/bug_report_templates with bundler 1.16 unless https://github.com/bundler/bundler/issues/6072 has fixed.

https://travis-ci.org/rails/rails/jobs/320121102#L1135-L1139

Reply:
I see. It's still failing with bundler-1.16.1. I will investigate after Ruby 2.5 releasing.

Reply:
I've just confirmed that https://github.com/bundler/bundler/issues/6072 has fixed in bundler-1.16.1.

```
% ruby action_controller_master.rb
Fetching https://github.com/rails/rails.git
Fetching gem metadata from https://rubygems.org/.............
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies....
Using rake 12.3.0
Using concurrent-ruby 1.0.5
Using i18n 0.9.1
Using minitest 5.10.3
Using thread_safe 0.3.6
Using tzinfo 1.2.4
Using activesupport 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using builder 3.2.3
Using erubi 1.7.0
Using mini_portile2 2.3.0
Using nokogiri 1.8.1
Using rails-dom-testing 2.0.3
Using crass 1.0.3
Using loofah 2.1.1
Using rails-html-sanitizer 1.0.3
Using actionview 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using rack 2.0.3
Using rack-test 0.8.2
Using actionpack 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using nio4r 2.1.0
Using websocket-extensions 0.1.3
Using websocket-driver 0.6.5
Using actioncable 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using globalid 0.4.1
Using activejob 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using mini_mime 1.0.0
Using mail 2.7.0
Using actionmailer 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using activemodel 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using arel 9.0.0
Using activerecord 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using activestorage 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using bundler 1.16.1
Using method_source 0.9.0
Using thor 0.20.0
Using railties 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Using sprockets 3.7.1
Using sprockets-rails 3.2.1
Using rails 5.2.0.beta2 from https://github.com/rails/rails.git (at master@7d06070)
Run options: --seed 12128

# Running:

I, [2017-12-22T18:30:25.004215 #51701]  INFO -- : Started GET "/" for 127.0.0.1 at 2017-12-22 18:30:25 +0900
.

Finished in 0.449111s, 2.2266 runs/s, 2.2266 assertions/s.
1 runs, 1 assertions, 0 failures, 0 errors, 0 skips
```

Reply:
Thanks for your confirmation!